### DISCUSSION TITLE
Large worlds support

### DISCUSSION THREAD

**Discussion by OldSnake22** [NVIDIA-Omniverse/PhysX#39] [Q&A] [answered]
Hello, 

I can see this in PhysX 5 release notes: 

"The Foundation types PxVec2, PxVec3, PxVec4, PxQuat, PxMat33, PxMat34, PxMat44 and PxTransform now have higher-accuracy implementations that use double instead of float. These are not currently used directly in the PhysX SDK but can be used by clients of the SDK if needed."

If I understand it, the engine does not yet provide large world support to avoid float precision issues when dealing with large worlds ? I'm a bit confused by this part "These are not currently used directly in the PhysX SDK but can be used by clients of the SDK if needed."

If not do, are you planning to add this feature in the near future ? 

Thank you !

**Comment by amoravanszky** [comment, marked as answer]
Yes, we have plans.  Or are at least planning to make plans.  It is a topic.  Don't ask when it is coming because I don't know yet.  :)

**Comment by iniside** [comment]
Are there still plans ? Or topic has been abandoned ?

**Comment by amoravanszky** [comment]
We have mostly been focusing on small to medium size simulations recently, so this didn't bubble up to the top of the priority list since the above exchange.  We are not actively working on it.
